subsys/bluetooth/host/audio/iso.c

```diff
@@ -207,6 +207,7 @@ void hci_le_cis_req(struct net_buf *buf)
 	bt_conn_unref(conn);
 
 	if (!iso) {
+		BT_DBG("Could not create and add ISO to conn %d", acl_handle);
```

Comment:
Wouldn't BT_WARN or BT_ERR be more appropriate?

Reply:
`BT_ERR` makes sense here, as it's due to missing allocations (out of memory). I see that in hci_core it seems that using `BT_ERR` is the way to go to be consistent.